docs/releasing.md

```diff
@@ -62,23 +62,27 @@ master.
 
 This step is not necessarily required if the alpine base image has
 recently been updated, but it is good to pick up any recent bug
-fixes. Updating the alpine base image is different to other packages
-and it must be performed on `x86_64` first:
+fixes. Updating the alpine base image is different to other packages.
+You must perform the arch-specific image builds, pushes and updates on each
+architecture first - these can be done in parallel, if you choose. When done,
+you then copy the updated `versions.<arch>` to one place, commit them, and
```

Comment:
This description does not match the the steps below where we still go from one arch for the initial build and then other arches with git forced pushes etc

Reply:
Good point. I updated the steps to reflect the above: build and push on each, collect the updated files, commit.